examples-gallery/two-pendulum-cart-variable-gallery.py

```diff
@@ -0,0 +1,311 @@
+"""
+Upright a double pendulum.
+==========================
```

Comment:
The title shoudl have `===` above and below it.

Reply:
clear, corrected